### ISSUE TITLE
It always download models from the web

### ISSUE THREAD

**Issue by itswyri** [ShapeNet/shapenet-viewer#21]
When I load some model or save screenshots by
> save model screenshots all 3dw <category>
my viewer always download some .kmz file from the web and it consumes a lot of time
although, I have registerd my shapeNetCore directory

The command "set loadFormat obj" seems to work, however, it works only one time.
How can I set my viewer to load some models from my hard drive?

**Comment by msavva** [comment]
It shouldn't be necessary to use the `set loadFormat obj` command as the obj format is the default when the local dataset has been successfully registered.  Can you please confirm that you don't get any error messages in the console after calling the `register shapeNetCore <path>` command?

**Comment by baudm** [comment]
I can confirm the same behavior. `register shapeNetCore <path>` does not show any error, but `save model screenshots all <category>` still makes the viewer download *.kmz files for each model.

**Comment by hajipour** [comment]
@baudm Please consider that you also need to extract all of the categories of the shapeNet ( e.g. 02691156)

**Comment by baudm** [comment]
@hajipour Yes I've downloaded the whole ShapeNetCore v1 (and even v2) archive. The viewer still downloads the models as if they don't exist in my local copy of ShapeNetCore.

**Comment by 0xBrandon** [comment]
This is still happening and I have most recent release, is there an ETA when this will be repaired?
